README.md

```diff
@@ -11,50 +15,31 @@ If you want to suggest a new feature for the C# or Visual Basic languages go her
 - [dotnet/vblang](https://github.com/dotnet/vblang) for VB-specific features
 - [dotnet/csharplang](https://github.com/dotnet/csharplang) for features that affect both languages
 
-## Contribute!
+### Contributing
 
-Some of the best ways to contribute are to try things out, file bugs, and join in design conversations. 
+All work on the C# and Visual Basic compiler happens directly on [GitHub](https://github.com/dotnet/roslyn). Both core team members and external contributors send pull requests which go through the same review process.
 
-### Questions
+If you are interested in fixing issues and contributing directly to the code base, a great way to get started is to ask some questions on [GitHub Discussions](https://github.com/dotnet/roslyn/discussions)! Then check out our [contributing guide](https://github.com/dotnet/roslyn/blob/master/docs/wiki/Contributing-Code.md) which covers the following:
 
-A great way to get started is to ask some questions!
-- Start with a question on [discussions](https://github.com/dotnet/roslyn/discussions)
-- You can also join in on the design discussions on [gitter](https://gitter.im/dotnet/roslyn) or [discord](http://aka.ms/discord-csharp-roslyn)
+- [Coding guidelines](https://github.com/dotnet/roslyn/blob/master/docs/wiki/Contributing-Code.md)
```

Comment:
Similar feedback here. The WIKI docs are out of date, should be focusing on the ones udner docs/contributing

Reply:
At the very least we should be putting all of our contrib docs that we're pointing users to in a single place